.github/workflows/ci.yml

```diff
@@ -8,15 +8,15 @@ on:
     branches: [ exercises-master ]
 
 jobs:
-  test_course:
+  validate_course:
```

Comment:
Renamed `job` from `test_course` to `validate_course` to be more specific.

A further change in `branch protection settings of repo` is required.